src/main/java/ch/njol/skript/expressions/ExprVehicle.java

```diff
@@ -38,102 +29,108 @@
 import ch.njol.skript.expressions.base.SimplePropertyExpression;
 import ch.njol.util.coll.CollectionUtils;
 
-/**
- * @author Peter Güttinger
- */
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.bukkit.event.vehicle.VehicleEnterEvent;
+import org.bukkit.event.vehicle.VehicleExitEvent;
+import org.eclipse.jdt.annotation.Nullable;
+import org.spigotmc.event.entity.EntityDismountEvent;
+import org.spigotmc.event.entity.EntityMountEvent;
+
 @Name("Vehicle")
-@Description({"The vehicle an entity is in, if any. This can actually be any entity, e.g. spider jockeys are skeletons that ride on a spider, so the spider is the 'vehicle' of the skeleton.",
-		"See also: <a href='#ExprPassenger'>passenger</a>"})
+@Description({
+	"The vehicle an entity is in, if any.",
+	"This can actually be any entity, e.g. spider jockeys are skeletons that ride on a spider, so the spider is the 'vehicle' of the skeleton.",
+	"See also: <a href='#ExprPassenger'>passenger</a>"
+})
 @Examples({"vehicle of the player is a minecart"})
 @Since("2.0")
 public class ExprVehicle extends SimplePropertyExpression<Entity, Entity> {
-	
-	static final boolean hasMountEvents = Skript.classExists("org.spigotmc.event.entity.EntityMountEvent");
-	
+
+	private static final boolean hasMountEvents = Skript.classExists("org.spigotmc.event.entity.EntityMountEvent");
```

Comment:
This can be removed, now that's we're 1.13+.
We should also switch to org.bukkit.event.entity.EntityMountEvent for 1.20+